build/resources/main/system.properties

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
there should be 1 such file for all project, why do we need this at all? and login and password should be stored in some enum, not in config.